tic_tac_toe/game.py

```diff
@@ -63,22 +95,60 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    # positions = [
```

Comment:
ahhhh you're too fast T___T I'm currently modifying it. we pushed first to make sure it worked with all Python versions